irodsConnector/permission.py

```diff
@@ -106,11 +106,10 @@ def set_permissions(self, perm: str, path: str, user: str = '',
                     self.data_man.collection_exists(path):
                 self.sess_man.irods_session.permissions.set(acl, recursive=recursive, admin=admin)
         except irods.exception.CAT_INVALID_USER as ciu:
```

Comment:
I think for consistency, `ciu` and `cia` should become `error`.

Reply:
Makes sense as long as they are only used locally and not needed again.